docs/release-notes/cloud/cloud-2023-04.md

```diff
@@ -0,0 +1,78 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud April 2023.
+included_jira_versions: ['2023.04']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.0.9
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.3.5
+---
+
+# Cloud April 2023
+
+These release notes are for the Codacy Cloud updates during April 2023.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   We’ve revamped the way you [manage your repositories](../../organizations/managing-repositories.md) on Codacy. You can now select which repositories you want to follow on Codacy, giving you more control over your dashboard for a cleaner, more personalized view. (PLUTO-336)
```

Comment:
For this release note I decided to reuse the "[announcement pop-up](https://www.figma.com/file/CrGB1jNcGMT8M6Xa72fNNZ/Follow-behavior-for-repository-listing?node-id=1653-24311&t=s8pGuKTD6idKA7NH-0)" that we display on the Codacy UI.